packages/nextjs/app/large-grants/[grantId]/_components/CurrentStage/NewStageModal/index.tsx

```diff
@@ -115,27 +110,18 @@ export const NewStageModal = forwardRef<HTMLDialogElement, NewStageModalProps>(
                         <FormTextarea
                           label="Description"
                           showMessageLength
-                          {...formMethods.register(`milestones.${index}.description`)}
-                          error={errors?.milestones?.[index]?.description?.message}
+                          {...getCommonOptions(`milestones.${index}.description`)}
```

Comment:
Not tested, but I think the same will happen here.

Reply:
fixed [8a64e4b](https://github.com/BuidlGuidl/ens-pg/pull/105/commits/8a64e4b5c711d9a4f090670a0749a1c4cc4c1fd6)

also removed min=1 for number input since when submitting form it showed only that item error